### DISCUSSION TITLE
Guideline for the CHANGELOG

### DISCUSSION THREAD

**Discussion by kikuomax** [buefy/buefy#4083] [General]
@wesdevpro 
I think we need a minimum guideline for the CHANGELOG. Here are some concerns that make me shy of writing it:
- `B` prefixed or not when mentioning a component; e.g., `BModal` or `Modal`?
- Should a headline show the issue or solution?
- Should changes in the `docs` package be included?
- Past or present tense?
- Should I refer to the issue or PR?

**Comment by wesdevpro** [comment]
> @wesdevpro I think we need a minimum guideline for the CHANGELOG. Here are some concerns that make me shy of writing it:
> 
> * `B` prefixed or not when mentioning a component; e.g., `BModal` or `Modal`?
> * Should a headline show the issue or solution?
> * Should changes in the `docs` package be included?
> * Past or present tense?
> * Should I refer to the issue or PR?

Where I would start is what "types" of users need the CHANGELOG? And why do they need the CHANGELOG? What benefits does the CHANGELOG bring to those user's needs? What needs to be communicated in the CHANGELOG to fill those needs?

What types of users does Buefy have?
- Mostly developers
  - Developers using Buefy to create and maintain their own projects.
  - The contributors and maintainers of Buefy.
